robot/description/urdf/caffeine.gazebo.xacro

```diff
@@ -93,6 +93,57 @@
             <!-- TODO: add Gaussian noise based off Garmin GPS -->
         </plugin>
     </gazebo>
+    
+    <!-- Gazebo multicamera plugin -->
+    <gazebo reference="${tf_prefix}/camera_link">
+        <sensor type="multicamera" name="stereo_camera">
+            <update_rate>${ZED_image_refresh}</update_rate>
+            <camera name="left">
+            <horizontal_fov>${ZED_image_hfov * (3.14/180)}</horizontal_fov>
+            <image>
+                <width>${ZED_image_width}</width>
+                <height>${ZED_image_height}</height>
+                <format>${ZED_image_format}</format>
+            </image>
+            <clip>
+                <near>0.02</near>
+                <far>300</far>
+            </clip>
+            </camera>
+            <camera name="right">
+            <pose>0 ${-ZED_camera_length} 0 0 0 0</pose>
+            <horizontal_fov>${ZED_image_hfov * (3.14/180)}</horizontal_fov>
+            <image>
+                <width>${ZED_image_width}</width>
+                <height>${ZED_image_height}</height>
+                <format>${ZED_image_format}</format>
+            </image>
+            <clip>
+                <near>0.02</near>
+                <far>300</far>
+            </clip>
+            </camera>
+            <plugin name="stereo_camera_controller" filename="libgazebo_ros_multicamera.so">
+                <cameraName>zed/zed_node</cameraName>
+                <alwaysOn>true</alwaysOn>
+                <updateRate>0</updateRate>
+                <cameraName></cameraName>
+                <imageTopicName>image_raw_color</imageTopicName>
```

Comment:
I believe it'd be better to put some of this stuff in constants (see how the LIDAR was done for example)